src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -403,89 +352,6 @@ final class HTTPClient: Client {
 		})
 	}
 	
-	// swiftlint:disable:next cyclomatic_complexity
-	func getDigitalCovid19Certificate(
-		registrationToken token: String,
-		isFake: Bool,
-		completion: @escaping DigitalCovid19CertificateCompletionHandler
-	) {
-		guard let request = try? URLRequest.dccRequest(
-				configuration: configuration,
-				registrationToken: token,
-				headerValue: isFake ? 1 : 0
-		) else {
-			completion(.failure(.urlCreationFailed))
-			return
-		}
-		
-		session.response(for: request, completion: { result in
-			switch result {
-			case let .success(response):
-				switch response.statusCode {
-				case 200:
-					guard let responseBody = response.body else {
-						Log.error("Error in response body: \(response.statusCode)", log: .api)
-						completion(.failure(.unhandledResponse(response.statusCode)))
-						return
-					}
-					do {
-						let decodedResponse = try JSONDecoder().decode(
-							DCCResponse.self,
-							from: responseBody
-						)
-						completion(.success(decodedResponse))
-					} catch {
-						Log.error("Failed to decode response json", log: .api, error: error)
-						completion(.failure(.jsonError))
```

Comment:
For this error you take now the default one of the RestService? Because I could not find this one in customError() in the Resource

Reply:
Exactly, just using the default one, and the error message is the same as both cases had the `.tryAgain` error text